app/_src/production/dp-config/cni.md

```diff
@@ -305,13 +305,10 @@ helm install --create-namespace --namespace {{site.mesh_namespace}} \
 
 {% endtabs %}
 
-### {{site.mesh_product_name}} CNI Logs
-
-Logs of the CNI plugin are available in `/tmp/kuma-cni.log` on the node and the logs of the installer are available via `kubectl logs`.
-
+{% if_version lte:2.1.x %}
 ## {{site.mesh_product_name}} CNI v2
 
-The v2 version of the CNI is using [kuma-net](https://github.com/kumahq/kuma-net/) engine to do transparent proxying.
+The CNI v2 is a rewritten and improved version of our original transparent-proxy.  
```

Comment:
```suggestion
The CNI v2 is a rewritten and improved version of original transparent-proxy.
```

Reply:
There's an article missing like this. What's wrong with our?

Reply:
I just did what `vale` suggested - `Try to avoid using first-person plural like 'our'.`